src/semantic-highlighting.ts

```diff
@@ -0,0 +1,168 @@
+import { BaseLanguageClient, ExtensionContext, State, StaticFeature, workspace } from 'coc.nvim';
+import { ClientCapabilities, NotificationType, Range, ServerCapabilities, TextDocumentClientCapabilities, VersionedTextDocumentIdentifier } from 'vscode-languageserver-protocol';
+
+// Contains the highlighting information for a specified line. Mirrors the
+// structure in the semantic highlighting proposal for LSP.
```

Comment:
Link to the proposal you're talking about - particularly since there are now two :-)

Reply:
semanticHighlighting docs has been removed from clangd website.

Reply:
It's not a clangd proposal: https://github.com/microsoft/vscode-languageserver-node/pull/367